src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  @include flex-column-centred;
+
+  max-width: $card-width;
```

Comment:
I would suggest changing it to max-width

Per task, you need 200px cards, while max-width does not guarantee the width
so `width:$card-width` look like cleaner solution here